test/infrastructure/docker/e2e/local-e2e.conf

```diff
@@ -0,0 +1,74 @@
+---
+images:
+# Using the Makefile found in the docker provider, run `make docker-build REGISTRY=gcr.io/k8s-staging-capi-docker`.
+- name: gcr.io/k8s-staging-capi-docker/capd-manager-amd64:dev
+  loadBehavior: mustLoad
+# Use `make docker-build REGISTRY=gcr.io/k8s-staging-cluster-api` to build these images.
+- name: gcr.io/k8s-staging-cluster-api/cluster-api-controller-amd64:dev
```

Comment:
There's no good way to do this right now. If we want to test using the master-release we should build a different config. The problem is that the `make docker-build` will build a single image and the released staging image is a manifest. So either we define the manifest here and spend a bunch of time building all images and then a manifest image or we define specific image architectures and use some other configuration for testing master images. I've found that 100% of the time I want to rebuild Cluster API managers from my local source and test those along with my local capd manager.

That's how I ended up on this solution. It's the fewest steps and least confusing path to run the tests locally.